src/main/kotlin/racing/service/GameMachine.kt

```diff
@@ -1,32 +1,18 @@
 package racing.service
 
+import racing.model.Car
 import racing.model.Racing
 
 object GameMachine {
-    private const val CAR_SYMBOL = "-"
-    private const val RESULT_SENTENCE = "실행 결과"
-    fun askCounting(question: String): Int {
-        println(question)
-        return readln().orEmpty().toInt()
-    }
 
-    fun printResult(carCount: Int, tryGameCount: Int) {
-        println(RESULT_SENTENCE)
-        play(carCount, tryGameCount)
+    fun getWinningCarName(racing: Racing): List<Car> {
+        val maxPosition = racing.cars.maxOf { it.position }
+        return racing.cars.filter { it.position == maxPosition }.toMutableList()
```

Comment:
`getWinningCarName`의 반환타입을 봤을 때는, `toMutableList` 함수는 제거하셔도 되지 않을까 싶어요. 🤔
```suggestion
        return racing.cars.filter { it.position == maxPosition }
```